scripts/test-react-native-setup.sh

```diff
@@ -13,7 +15,7 @@ buck build ReactAndroid/src/main/java/com/facebook/react/shell
 
 echo "Build React Native via Gradle"
 yarn install
-./gradlew --no-daemon :ReactAndroid:packageReactNdkLibsForBuck
+./gradlew :ReactAndroid:packageReactNdkLibsForBuck
 
 echo "Assemble RNTester app"
-./gradlew --no-daemon :packages:rn-tester:android:app:assembleRelease
```

Comment:
I would keep the `--no-daemon` flag here. Any reason for removing it?

Reply:
For build speed, but looks github action still timeout with.

Reply:
IMHO the build speed we gain out of the daemon is not worth the loss in isolation